### ISSUE TITLE
[BUG] "Handler not found for view SkiaSharp.Views.Maui.Controls.SKGLView."

### ISSUE THREAD

**Issue by ToolmakerSteve** [mono/SkiaSharp#2860]
### Description

Target Windows Machine, get Unhandled Exception:  

"Handler not found for view SkiaSharp.Views.Maui.Controls.SKGLView."

-- Other details --

.csproj targetframework: net8.0-windows10.0.19041.0

VS Community 17.9.7 on Windows 11 Home

Tested with Microsoft.Maui.Controls 8.0.40 +

SkiaSharp.Views & SkiaSharp.Views.Maui Controls:

* latest stable 2.88.8, then tried
* latest preview 3.0.0-preview.3.1

* Did not try on earlier versions of Maui or these SkiaSharp nugets.
* Before Maui, we used OpenTK.GLWpfControl with SkiaSharp.Views.WPF under .Net Framework. That older version of our app builds and runs on this PC without problems.

### Code

Maui App template. Modified to add SKGLView. Removed AppShell and other unneeded stuff.

[ToolmakerSteve/MauiAppWithSKGLView](https://github.com/ToolmakerSteve/MauiAppWithSKGLView)

```cs
        public MainPage()
        {
            InitializeComponent();

            MainThread.BeginInvokeOnMainThread(async () =>
            {
                await Task.Delay(2000);
                // UNCOMMENT these lines to test adding SKGLView in code.
                //var skv = new SKGLView() { WidthRequest = 300, HeightRequest = 300 };
                // Below line: Exception: "Handler not found for view SkiaSharp.Views.Maui.Controls.SKGLView."
                //layout.Children.Add(skv);
            });
        }
```
You can also share some XAML:
```xaml
<ContentPage
    xmlns:sk="clr-namespace:SkiaSharp.Views.Maui.Controls;assembly=SkiaSharp.Views.Maui.Controls"
    ...

    <VerticalStackLayout x:Name="layout" Padding="30,0" Spacing="25">
        <!-- Exception: "Handler not found for view SkiaSharp.Views.Maui.Controls.SKGLView."  -->
        <sk:SKGLView WidthRequest="500" HeightRequest="300" BackgroundColor="Green" />
    </VerticalStackLayout>
```

### Expected Behavior

A Window containing an SKGLView.

### Actual Behavior

Unhandled Exception:  

"Handler not found for view SkiaSharp.Views.Maui.Controls.SKGLView."

### Version of SkiaSharp

Other (Please indicate in the description)

### Last Known Good Version of SkiaSharp

Other (Please indicate in the description)

### IDE / Editor

Visual Studio (Windows)

### Platform / Operating System

Windows

### Platform / Operating System Version

Windows 11 Home.

Did not test elsewhere.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by mattleibow** [comment]
I downloaded your sample and noticed the `.UseSkiaSharp()` line is missing from your MauiPRogram.cs:

![image](https://github.com/mono/SkiaSharp/assets/1096616/d9f0b7c8-2936-4c0c-9d6c-92553f6e8e48)

**Comment by ToolmakerSteve** [comment]
> I downloaded your sample and noticed the `.UseSkiaSharp()` line is missing from your MauiProgram.cs:

That fixes it. I completely forgot about that.

Sorry about the false alarm.

**Comment by mattleibow** [comment]
I need to put this in the docs... Not sure if there are any docs for setting up a maui with skia app.

**Comment by tmijieux** [comment]
Indeed the latest stable nuget  2.88.8 still has this problem (as we can see in the source code
https://github.com/mono/SkiaSharp/blob/release/2.88.8/source/SkiaSharp.Views.Maui/SkiaSharp.Views.Maui.Controls/AppHostBuilderExtensions.cs#L25
)
I currently had to switch the UseSkiaSharp(bool)overload  with true argument if i wanted it to work on ios with latest nuget(2.88.8) version

the doc i found on this was that:
https://learn.microsoft.com/en-us/dotnet/maui/migration/skiasharp?view=net-maui-8.0#initialize-skiasharp
 (as i was migrating  from a xamarin application, this is what I naturally  found)
 
 This thing is , i think the doc will be ok by the time you will release the next version...

**Comment by ToolmakerSteve** [comment]
@mattleibow - is there a plan to backport the Maui SKGLView fix on Windows to 2.88.x?

Updating my client from .Net Framework 4.5 to .Net 8.

We've successfully updated on iOS and Android.  
(WITHOUT Maui; these are "platform native" .Net 8 apps, with GUI entirely rendered via SkiaSharp. Plus some OpenGL.)

On Windows .Net 8, SkiaSharp 2.88.8 or .9, we've got everything EXCEPT OpenGL working. Using Maui SKCanvasView.

I made a new Maui app. Added SkiaSharp 2.88.9. Remembered to do "UseSkiaSharp()". Same handler exception.

Updating to SkiaSharp 3.118.0.0 preview, it works.  BUT this app is used in a production situation where failures must be very rare. So I can't use anything that isn't considered a stable release.
